src/pineko/evolve.py

```diff
@@ -172,21 +190,37 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
             "are you sure that's what you want?"
         )
 
-    with open(card_path, "w", encoding="UTF-8") as f:
-        yaml.safe_dump(operators_card, f)
-        f.write(f"# {pineko_version=}")
+    # For hardonic obs we might need to dump 2 eko cards
+    operators_card["configs"]["polarized"] = eko1 == "polPDF"
+    if eko1 == eko2:
+        with open(card_path, "w", encoding="UTF-8") as f:
+            yaml.safe_dump(operators_card, f)
+            f.write(f"# {pineko_version=}")
```

Comment:
split these 3 lines into a function and reuse

Reply:
actually, also the `card_name` adjustment should be part of that

Reply:
I'm not sure about adding `_eko1` also to the default name, I think this way is more
backward compatible.

Reply:
just take `suffix` as argument and then `if suffix != "": change()`

Reply:
done in 3c107ee